### ISSUE TITLE
Onnx 

### ISSUE THREAD

**Issue by Volition0451** [C0untFloyd/roop-unleashed#39]
**Describe the bug**
I get multiple "Failed to allocate memory for requested buffer of size 134217728" upon starting to process a video, after which the video is processed for a seemingly random number of frames before it stops due to an "Exception in tkinter callback". I'm fairly certain I've installed everything correctly.

The entire exception trace is given in screenshots. You can see that 25% had been processed post the initial exceptions before it stopped. It works well with CPU, but is absurdly slow (as expected)

**Details**
What OS are you using?
- [ ] Linux
- [ ] Linux in WSL
- [O] Windows 
- [ ] Mac

Are you try to use a GPU?
- [ ] No. I am not using the `---gpu` flag
- [O] NVIDIA
- [ ] AMD
- [ ] Intel
- [ ] Mac

**Screenshots**
25% can be seen in the middle of the first SS, and at the top of the second SS. The output ends at the second screenshot's last line.

![image](https://github.com/C0untFloyd/roop-unleashed/assets/136564385/3a0a932d-946b-4d3a-ba5f-a07131251035)

![image](https://github.com/C0untFloyd/roop-unleashed/assets/136564385/35d7f700-36d0-49e7-b8ca-3c8642b2aefa)

**Sanity Check**
- [O] I have the latest code from the github repository
- [O] I have followed the installation guide

**Comment by eniora** [comment]
Have you guys tried running roop with **--execution-threads 2** command line argument?
You can start with 2 (or even 1) and keep raising until you get the problem then lower it. I remember reading that people with low VRAM GPUs (less than 8gb) trying this and it helped them.

**Comment by Volition0451** [comment]
Yes, this worked for me. I guess each thread needs to be allocated memory which isn't available. Thanks!
